client/src/components/atoms/style/StepNumber.scss

```diff
@@ -0,0 +1,20 @@
+.step-number {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 45px;
+    height: 45px;
+    border: 1px solid $stepper-border-color;
```

Comment:
minor color difference: in the figma design the border is `#B7B7B7` but `$stepper-border-color` is defined as `#C4C4C4`